airflow/models/all_models/__init__.py

```diff
@@ -0,0 +1,154 @@
+        return BaseOperatorLink
+    if name == 'Connection':
+        from airflow.models.connection import Connection
+        return Connection
+    if name == 'DAG':
+        from airflow.models.dag import DAG
+        return DAG
+    if name == 'DagModel':
+        from airflow.models.dag import DagModel
+        return DagModel
+    if name == 'DagTag':
+        from airflow.models.dag import DagTag
+        return DagTag
+    if name == 'DagBag':
+        from airflow.models.dagbag import DagBag
+        return DagBag
+    if name == 'DagPickle':
+        from airflow.models.dagpickle import DagPickle
+        return DagPickle
+    if name == 'DagRun':
+        from airflow.models.dagrun import DagRun
+        return DagRun
+    if name == 'ImportError':
+        from airflow.models.errors import ImportError
+        return ImportError
+    if name == 'KubeResourceVersion':
+        from airflow.models.kubernetes import KubeResourceVersion
+        return KubeResourceVersion
+    if name == 'KubeWorkerIdentifier':
+        from airflow.models.kubernetes import KubeWorkerIdentifier
+        return KubeWorkerIdentifier
+    if name == 'Log':
+        from airflow.models.log import Log
+        return Log
+    if name == 'Pool':
+        from airflow.models.pool import Pool
+        return Pool
+    if name == 'SerializedDagModel':
+        from airflow.models.serialized_dag import SerializedDagModel
+        return SerializedDagModel
+    if name == 'SkipMixin':
+        from airflow.models.skipmixin import SkipMixin
+        return SkipMixin
+    if name == 'SlaMiss':
+        from airflow.models.slamiss import SlaMiss
+        return SlaMiss
+    if name == 'TaskFail':
+        from airflow.models.taskfail import TaskFail
+        return TaskFail
+    if name == 'TaskInstance':
+        from airflow.models.taskinstance import TaskInstance
+        return TaskInstance
+    if name == 'clear_task_instances':
+        from airflow.models.taskinstance import clear_task_instances
+        return clear_task_instances
+    if name == 'TaskReschedule':
+        from airflow.models.taskreschedule import TaskReschedule
+        return TaskReschedule
+    if name == 'Variable':
+        from airflow.models.variable import Variable
+        return Variable
+    if name == 'XCOM_RETURN_KEY':
+        from airflow.models.xcom import XCOM_RETURN_KEY
+        return XCOM_RETURN_KEY
+    if name == 'XCom':
+        from airflow.models.xcom import XCom
+        return XCom
+    raise AttributeError(f"module {__name__} has no attribute {name}")
+
+
+# This is never executed, but tricks static analyzers (PyDev, PyCharm,
+# pylint, etc.) into knowing the types of these symbols, and what
+# they contain.
+STATICA_HACK = True
+globals()['kcah_acitats'[::-1].upper()] = False
+if STATICA_HACK:  # pragma: no cover
+    from airflow.models.base import ID_LEN
+    from airflow.models.base import Base
+    from airflow.jobs.base_job import BaseJob
```

Comment:
Including `airflow.jobs.base_job` in `airflow.models.all_models` feels a little out of place here. Maybe the name/location of `airflow.models.all_models` should be changed?

Reply:
Maybe something like `airflow.utils.sqla` ?

Reply:
The thing is that BaseJob IS a DB model :).